packages/atomic/src/components/common/result-list/styles/mixins.pcss

```diff
@@ -150,18 +150,27 @@
         &:hover {
           border: 1px solid var(--atomic-neutral);
           box-shadow: 0px 10px 25px var(--atomic-neutral);
+          cursor: pointer;
         }
       }
     }
   }
 
-  [part='result-list-grid-clickable'] {
-    content: '';
+  [part='result-list-grid-clickable']:focus-visible {
```

Comment:
Due to this change, I think the final commit should be a `feat` instead of a `fix`.

Like "feat(atomic): added skip link to grid results".